wormhole/file_transport.go

```diff
@@ -232,10 +239,16 @@ func (t *fileTransport) connectDirect(otherTransit *transitMsg) (net.Conn, error
 
 	var conn net.Conn
 
+	connectTimeout := time.After(5 * time.Second)
+
```

Comment:
same as above. The timeout and the `select()` statement should not be added back.